### ISSUE TITLE
Webhooks UI still having problems.

### ISSUE THREAD

**Issue by richburdon** [docker/hub-feedback#336]
1. The History button doesn't do anything that is apparent.
2. After pressing Edit, the red "X" button generates the following JS error:
   DELETE https://hub.docker.com/v2/repositories/alienlabs/nx-frontend/webhooks/5021/hooks/7045/ 404 (NOT FOUND)
3. The Save button does nothing.
4. The Delete Pipeline button generates the following error:
   DELETE https://hub.docker.com/v2/repositories/alienlabs/nx-frontend/webhooks/5021/ 404 (NOT FOUND)
5. Webhooks don't seem to get triggered.

So, basically nothing seems to work.

**Comment by mariusGundersen** [comment]
1. The history button opens a list, where each item has an id, clicking the id opens another list. No matter which id is clicked, the same list is opened. This is confusing
2. The red X changes to "Deleted" when clicked, and clicking the "Deleted" button changes it back to an X. Clicking the X again flashes "Deleting", but it returns to an X again. This is confusing
3. The save button does nothing
4. The x in the top right of the box does nothing

In general this entire interface is very confusing and doesn't seem to work well. Clicking buttons don't trigger actions (or at least it doesn't look to the user like it is triggering an action). Many of the buttons only have icons with no description, so it is often not clear what they will do if you click on them.

**Comment by ChristopherBiscardi** [comment]
There are fixes coming down the pipeline for this interface.

**Comment by vayasin** [comment]
Any updates on this issue?
